src/Microsoft.SqlTools.ServiceLayer/Security/SecurityService.cs

```diff
@@ -169,6 +184,152 @@ await requestContext.SendResult(new ResultStatus()
             });
         }
 
+        internal async Task HandleUpdateLoginRequest(UpdateLoginParams parameters, RequestContext<object> requestContext)
+        {
+            ConnectionInfo connInfo;
+            string ownerUri;
+            contextIdToConnectionUriMap.TryGetValue(parameters.ContextId, out ownerUri);
```

Comment:
the final design will be clone the connect during init and close on dispose.  But this is fine for initial PRs to reuse passed in connection, which is likely from OE or somewhere else.